### PR TITLE
Change ManagementContract Name

### LINKED ISSUE #2132: ManagementContract.Name should be a valid code identifier
ManagementContract.Name is currently "Neo Contract Management". The name should not have spaces so that we can generate valid code type names from the manifest. Additionally, there's no need for the word "Neo" in the name.

### PR DESCRIPTION
**Pull request by devhawk**
fixes #2132

### CODE REVIEW

File: src/neo/SmartContract/Native/ManagementContract.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ namespace Neo.SmartContract.Native
 {
     public sealed class ManagementContract : NativeContract
     {
-        public override string Name => "Neo Contract Management";
+        public override string Name => "ContractManagement";
```
Comment (shargon) [inline src/neo/SmartContract/Native/ManagementContract.cs:15]: rename the class too?

Comment (devhawk) [reply src/neo/SmartContract/Native/ManagementContract.cs:15]: Personally, I don't care. But yeah, I guess it should match. Given the naming of other native contracts, "ContractManagement" seems like a better choice than "ManagementContract", but as long as the name is a legal identifier I 100% do not care.

**Comment by erikzhang** [comment]
@devhawk @shargon Please check #2138

**Comment by devhawk** [comment]
> @devhawk @shargon Please check #2138

That works. I'll close this PR once I get a chance to pull those changes into the monorepo

**Pull request closed without merging**